rankboost.py

```diff
@@ -203,8 +205,14 @@ def get_mutations(mutations):
             temp_mutation.append(re.sub('[0-9]+', '>', mut))
             prev_pos = curr_pos
         out_muts.append(''.join(temp_mutation))
-    assert len(set(out_muts)) == 1
-    return out_muts[0]
+    if len(set(out_muts)) > 1:
```

Comment:
Ada could you please document these lines for future reference? I'll accept asap after